man/man8/zfs-rollback.8

```diff
@@ -30,25 +29,20 @@
 .\" Copyright 2018 Nexenta Systems, Inc.
 .\" Copyright 2019 Joyent, Inc.
 .\"
-.Dd June 30, 2019
+.Dd May 27, 2021
 .Dt ZFS-ROLLBACK 8
 .Os
+.
 .Sh NAME
 .Nm zfs-rollback
-.Nd Roll back the given dataset to a previous snapshot.
+.Nd roll ZFS dataset back to snapshot
```

Comment:
maybe: "rollback a ZFS dataset to a snapshot"

Reply:
as above